dds_cli/__main__.py

```diff
@@ -1430,6 +1384,123 @@ def fix_project_access(click_ctx, email, project):
         sys.exit(1)
 
 
+# INFO ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ INFO #
+@project_group_command.group(name="info", no_args_is_help=True)
+@click.pass_obj
+def project_info(_):
+    """Manage project information.
+
+    Display or change the information of a project.
+
+    Displaying the project information is available for all user roles. Changing the project information
+    is limited to Unit Admins and Personnel.
+    """
+
+
+# -- dds project info display -- #
+@project_info.command(name="display")
+# Options
+@click.option(
+    "--project",
+    "-p",
+    required=True,
+    type=str,
+    help="The ID of the project.",
+)
+@click.pass_obj
+def display_project_info(
+    click_ctx,
+    project,
+):
+    """Display information about a specific project.
+
+    Usable by all user roles.
+
+    \b
+    The following information should be displayed:
+    - Project ID
+    - Creator
+    - Status
+    - Date updated
+    - Size
+    - Title
+    - Description
+    """
+    try:
+        with dds_cli.project_info.ProjectInfoManager(
+            project=project,
+            no_prompt=click_ctx.get("NO_PROMPT", False),
+            token_path=click_ctx.get("TOKEN_PATH"),
+        ) as get_info:
+            get_info.show_project_info()
+    except (
+        dds_cli.exceptions.APIError,
+        dds_cli.exceptions.AuthenticationError,
+        dds_cli.exceptions.DDSCLIException,
+        dds_cli.exceptions.ApiResponseError,
+        dds_cli.exceptions.ApiRequestError,
+    ) as err:
+        LOG.error(err)
+        sys.exit(1)
+
+
+# -- dds project info change -- #
+@project_info.command(name="change", no_args_is_help=True)
+# Options
+@click.option(
+    "--project",
+    "-p",
+    required=True,
+    type=str,
+    help="The ID of the project.",
+)
+@click.option(
+    "--title",
+    "-t",
+    required=False,
+    help="Change the title of the project.",
+)
+@click.option(
+    "--description",
+    "-d",
+    required=False,
+    help="Change the description of the project.",
+)
+@click.option(
+    "--principal-investigator",
+    "-pi",
+    required=False,
+    help="Change the PI of the project.",
+)
+@click.pass_obj
+def change_project_info(click_ctx, project, title, description, principal_investigator):
+    """Change project information.
+
+    Not available for Researchers, unless they are marked as Project Owner for a specific project.
+
+    \b
+    Use the `--title` option to change the Title of a project.
+    Use the `--description` option to change the Description of a project.
+    Use the `"--principal-investigator"` option to change the PI of a project.
+    """
```

Comment:
Add a check here to make sure that at least one of the options are used. Atm this is what happens
```bash
/code # dds project info change -p project_1 
[...]
ERROR    ⚠️ Failed to update project info: Required data missing from request! ⚠️         
```

So add a message such as "Please specify which information you would like to change: '--title', '--description' or/and '--principal-investigator'." or something like this

Reply:
Also, seems like there's currently an error when only trying to change the title for example
```bash
/code # dds project info change -p project_1 --title testing
[...]
ERROR    ⚠️ Failed to update project info: The PI email is invalid ⚠️      
```

Reply:
´/code # dds project info change -p project_1 --title testing
[...]
ERROR    ⚠️ Failed to update project info: The PI email is invalid ⚠️      `
I cannot reproduce this